playground/infrastructure/helm-playground/templates/deployment-frontend.yml

```diff
@@ -34,6 +34,25 @@ spec:
        imagePullPolicy: {{ .Values.image.pullPolicy }}
        ports:
         - containerPort: 8080
+       livenessProbe:
+         httpGet:
+           path: /liveness
+           port: 8080
+         initialDelaySeconds: 3
+         periodSeconds: 3
+       readinessProbe:
+         httpGet:
+           path: /readiness
+           port: 8080
```

Comment:
Is it possible for the `8080` to be acquired from a helm variable to populate `ports.containerPort`, `livenessProbe.httpGet.port`, and `readinessProbe.httpGet.port`?

Reply:
Fixed. Everything was moved to the values.yaml file